conf/nomad/presto_standalone.hcl

```diff
@@ -10,10 +10,13 @@ job "${nomad_job_name}" {
     healthy_deadline  = "12m"
     progress_deadline = "15m"
     auto_revert       = true
-    auto_promote      = true
+  %{ if use_canary }
     canary            = 1
+    auto_promote      = true
+  %{ endif }
     stagger           = "30s"
   }
```

Comment:
```suggestion
  %{ if use_canary }
    canary            = 1
    auto_promote      = true
    auto_revert       = true
  %{ endif }
    stagger           = "30s"
  }
```

Reply:
Ok, it didn't give any errors so didn't think of it! I'll commit your suggestion.